docs/csaf.md

```diff
@@ -0,0 +1,242 @@
+# Red Hat CSAF Files
+
+## CSAF Overview
+The [Common Security Advisory Framework (CSAF)](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html) 
+was originally published as an open standard by OASIS Open in November 2022. CSAF files provide a structured, 
+machine-readable way of representing and sharing security advisory information across all software and hardware providers. 
+ 
+Red Hat's CSAF files are always associated with RHSA and a given security advisory may include one or more product 
+version(s) and one or more components, depending on the product type and update scope. The RHSA itself can also 
+include updates to address one or more vulnerabilities. Red Hat’s CSAF files are publicly available per RHSA 
+[here](security.access.redhat.com/data).
+
+## Document Structure
+The CSAF standard requires three main sections: document metadata, a product tree and vulnerability metadata.
+The full document structure, without values, can be found
+[here](https://github.com/RedHatProductSecurity/security-data-guidelines/blob/csaf-vex-guidelines/docs/csaf.json).
+
+The following sections break down the information included in VEX documents using the
+[RHSA]() as an example.
+
+### Document Metadata 
+The `document` section contains general information about the published document itself including CVE severity, vendor,
+published date and revision history.
+
+The `aggregate_severity.text` object displays the general CVE severity:
+
+```
```

Comment:
If you add `json` right after the three backticks, it'll add syntax highlighting for the code block.